hmpo-lev/email/messages/messages_en.properties

```diff
@@ -0,0 +1,34 @@
+emailVerificationSubject=Welcome to LEV
+passwordResetSubject=Reset your LEV password
+emailVerificationBody=Hi {1} {2},\n\n\
+We have created an account for you with the Life Event Verification (LEV) service. Click the following link to confirm your email address and activate your account:\
+\n\n{3}\n\n\
```

Comment:
Should this me something like?:
```
<a href={3}>{3}</a>

Reply:
This is text only as a fallback in case the html version has errors, so I'd say no. 

Although the HTML version of the service links could be separated into several variables and implemented as you've suggested. I'm not sure of the value in it though...

Reply:
Okay.

If it was HTML I'd suggest it to handle the case when the text wouldn't automatically be made into a link.